uber/templates/preregistration/form.html

```diff
@@ -91,7 +95,7 @@
         <label for="tables" class="col-sm-2 control-label">Tables</label>
         <div class="col-sm-6">
             <select class="form-control" name="tables">
-                {% options c.TABLE_OPTS group.tables %}
+                {% int_options 1 4 group.tables %}
             </select>
```

Comment:
Yeah, we have it go from 0 to 9 in the admin interface.

Ironically, Anthrocon had these tied to options (notably TABLE_OPTS and ADMIN_TABLE_OPTS) but Eli took that out during the refactor. :P